src/pip/_internal/cli/cmdoptions.py

```diff
@@ -764,6 +764,19 @@ def _handle_no_cache_dir(
     help="Check the build dependencies when PEP517 is used.",
 )
 
+parallel_downloads: Callable[..., Option] = partial(
+    Option,
+    "--parallel-downloads",
+    dest="parallel_downloads",
+    type="int",
+    metavar="n",
+    default=None,
```

Comment:
I wonder if it’s easier to use 0 as the default instead of None. This would make checks below a bit simpler.

By the way, what’s the difference between setting this to 1 and not at all? We should probably add something in documentation about this.

Reply:
If the option is not set then in `pip._internal.network.session` self.parallel_downloads gets set to 1. So there isn't really a difference between setting it to 1 and not at all. I wasn't sure if it would be confusing setting parallel_downloads to 0 or 1 by default to indicate no parallel downloads. At the time of writing I thought a default of none to indicate no parallel_downloads made sense, but I'm happy to  change it if you think 0 is better.

And thinking about it a bit more, it might make more sense to make 2 the minimum number of parallel downloads, what do you think?

Reply:
I think it’s beneficial to have a value that simply means the default. This makes the CLI easier to interface, e.g. when writing a Bash script to conditionally switch parallel downloads on and off. So I’d want this to allow 1 and up.

Reply:
That makes sense to me.  I'll update it to make 1 the default value.